src/config/webpack.config.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const path = require('path')
+const webpack = require('webpack')
+const merge = require('webpack-merge')
+const {fromRoot, pkg, paths, getLibraryName} = require('../utils')
+const userConfig = require('./user')()
+
+const base = (env = {production: true}, argv) => {
```

Comment:
default args are incompatible with early versions of node 4

Reply:
That's fine, we don't support node.js 4 anymore.